composer.json

```diff
@@ -26,13 +26,14 @@
         "symfony/validator": "5.0.*",
         "symfony/webpack-encore-bundle": "^1.7.3",
         "symfony/yaml": "^5.0.5",
+        "theofidry/alice-data-fixtures": "1.1.2",
         "twig/extra-bundle": "^3.0.3",
-        "twig/twig": "^3.0.3",
-        "theofidry/alice-data-fixtures" : "1.1.2"
+        "twig/twig": "^3.0.3"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.16.1",
         "hautelook/alice-bundle": "^2.7.2",
+        "nelmio/api-doc-bundle": "^3.6",
```

Comment:
Die Dokumentation zur Api darf doch öffentlich sein oder nicht?

Reply:
@bartrail @gherkins 
Was meint ihr? Soll die API doc öffentlich sein, oder nur im dev-Environment.
Ist ja für den Moment erstmal keine API, die öffentlich genutzt werden kann...

Reply:
jo deswegen vielleicht lieber nur in dev, oder?